azext_iot/operations/dps.py

```diff
@@ -18,36 +18,41 @@
 from azext_iot.common.utility import compute_device_key
 from azext_iot.operations.generic import _execute_query
 from azext_iot._factory import SdkResolver
-from azext_iot.sdk.dps.models.individual_enrollment import IndividualEnrollment
-from azext_iot.sdk.dps.models.custom_allocation_definition import (
+from azext_iot.sdk.dps.service.models.individual_enrollment import IndividualEnrollment
+from azext_iot.sdk.dps.service.models.custom_allocation_definition import (
     CustomAllocationDefinition,
 )
-from azext_iot.sdk.dps.models.attestation_mechanism import AttestationMechanism
-from azext_iot.sdk.dps.models.tpm_attestation import TpmAttestation
-from azext_iot.sdk.dps.models.symmetric_key_attestation import SymmetricKeyAttestation
-from azext_iot.sdk.dps.models.x509_attestation import X509Attestation
-from azext_iot.sdk.dps.models.x509_certificates import X509Certificates
-from azext_iot.sdk.dps.models.x509_certificate_with_info import X509CertificateWithInfo
-from azext_iot.sdk.dps.models.initial_twin import InitialTwin
-from azext_iot.sdk.dps.models.twin_collection import TwinCollection
-from azext_iot.sdk.dps.models.initial_twin_properties import InitialTwinProperties
-from azext_iot.sdk.dps.models.enrollment_group import EnrollmentGroup
-from azext_iot.sdk.dps.models.x509_ca_references import X509CAReferences
-from azext_iot.sdk.dps.models.reprovision_policy import ReprovisionPolicy
-from azext_iot.sdk.dps.models import DeviceCapabilities
-from azext_iot.sdk.dps.models import (
+from azext_iot.sdk.dps.service.models.attestation_mechanism import AttestationMechanism
+from azext_iot.sdk.dps.service.models.tpm_attestation import TpmAttestation
+from azext_iot.sdk.dps.service.models.symmetric_key_attestation import (
+    SymmetricKeyAttestation,
+)
+from azext_iot.sdk.dps.service.models.x509_attestation import X509Attestation
+from azext_iot.sdk.dps.service.models.x509_certificates import X509Certificates
+from azext_iot.sdk.dps.service.models.x509_certificate_with_info import (
+    X509CertificateWithInfo,
+)
+from azext_iot.sdk.dps.service.models.initial_twin import InitialTwin
+from azext_iot.sdk.dps.service.models.twin_collection import TwinCollection
+from azext_iot.sdk.dps.service.models.initial_twin_properties import (
+    InitialTwinProperties,
+)
+from azext_iot.sdk.dps.service.models.enrollment_group import EnrollmentGroup
+from azext_iot.sdk.dps.service.models.x509_ca_references import X509CAReferences
+from azext_iot.sdk.dps.service.models.reprovision_policy import ReprovisionPolicy
+from azext_iot.sdk.dps.service.models import DeviceCapabilities
+from azext_iot.sdk.dps.service.models import (
     ProvisioningServiceErrorDetailsException,
 )  # TODO: Regen SDK
```

Comment:
This same general error type persisted with the DPS SDK regen right? It wasn't changed to `CloudError` like other services? We can remove the TODO if `ProvisioningServiceErrorDetailsException` is thrown across the board for service issues with the `2019-03-31` version.